backend/de.metas.externalsystem/src/main/java/de/metas/externalsystem/export/pporder/ExportPPOrderToExternalSystem.java

```diff
@@ -55,7 +55,7 @@ public abstract class ExportPPOrderToExternalSystem extends ExportToExternalSyst
 {
 	private static final Logger logger = LogManager.getLogger(ExportPPOrderToExternalSystem.class);
 
-	private final IPPOrderDAO ppOrderDAO = Services.get(IPPOrderDAO.class);
+	protected final IPPOrderDAO ppOrderDAO = Services.get(IPPOrderDAO.class);
```

Comment:
Y protected?

Reply:
to be available in `ExportPPOrderToLeichMehlService` which extends `ExportPPOrderToExternalSystem`